### ISSUE TITLE
Bad link in Lore docs

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4241]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)| @khorn reported|
|-|-|
|Trac ID|trac#4241|
|Type|defect|
|Created|2010-01-28 20:31:12Z|

`/lore/howto/extend-lore` references `/core/plugin`, but that file does not exist.

Should reference `/core/howto/plugin`.

Attachments:

* [4241_bad_link_fix.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/858/858011a0f35faf85ad57bc7e1190038c4f36e4b2/5b0282cf5baa3ac0a141095add2b1bc84704d6b9.patch) (684 bytes) - added by khorn on 2010-01-28 20:34:15Z - Fix for bad link

<details><summary>Searchable metadata</summary>

```
trac-id__4241 4241
type__defect defect
reporter__khorn khorn
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__lore lore
keywords__documentation documentation
time__1264710672000000 1264710672000000
changetime__1264717209000000 1264717209000000
version__None None
owner__ 
cc__khorn cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_2"></a>|@khorn removed owner|
|-|-|

Attached patch should fix.  Please review.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra set owner to @khorn|
|-|-|

Please merge, thanks.

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_4"></a>|@khorn set status to `closed`|
|-|-|

(In [28088]) Merge: Fix broken link in extend-lore.xhtml

Author: khorn
Reviewers: thijs
Fixes: [#4241](https://github.com/twisted/twisted/issues/4241)

Fix broken link in /doc/lore/howto/extend-lore.xhtml

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
